cal/cal/migrations/0046_auto_20170107_2152.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
```

Comment:
What is with the new migration file? Is this because you renamed a bunch of ColorCategory stuff?